src/components/Navigation/Navigation.jsx

```diff
@@ -78,7 +80,7 @@ export default class Navigation extends React.Component {
 
         { Links.filter(link => this._isActive(link) && link.children).map(link => (
           <div className="navigation__bottom" key={ link.title }>
-            <Container className="navigation__inner">
+            <nav className="container navigation__inner">
```

Comment:
What is the reason of this change?

Reply:
Ah this one. So when using a screenreader there is a way quickly jump to navigations. By using <nav></nav> we add a navigation landmark.